.pre-commit-config.yaml

```diff
@@ -20,6 +20,11 @@ repos:
       - id: "check-merge-conflict"
       - id: "check-yaml"
         args: ["--unsafe"]
+        exclude: |
+            (?x)^(
+                custom_components/ui_lovelace_minimalist/installation/.*\.yaml|
```

Comment:
Why do you add these folders/data to be ignored by "chack-yaml"?

Reply:
Because of Jinja templating and this check not liking it. 😅

Reply:
Might think of changing it to extension based. But requires some more work.

Reply:
Ahh, good to know.
What's exactly the problem with jinja templating?
What error is it throwing?

Reply:
```
while scanning for the next token
found character that cannot start any token
  in "custom_components/ui_lovelace_minimalist/lovelace/views/main/rooms/room.yaml", line 5, column 2
while scanning for the next token
found character that cannot start any token
  in "custom_components/ui_lovelace_minimalist/lovelace/views/main/01_home.yaml", line 8, column 6
````